src/docfx/build/legacy/manifest/LegacyManifest.cs

```diff
@@ -33,7 +33,7 @@ public static void Convert(Docset docset, Context context, Dictionary<Document,
 
                         var output = new LegacyManifestOutput
                         {
-                            MetadataOutput = document.IsSchemaData || document.ContentType == ContentType.Resource
+                            MetadataOutput = document.IsData || document.ContentType == ContentType.Resource
```

Comment:
Seems IsData is mostly used in legacy, can we remove it from Document and directly use TemplateEngine.IsData?

Reply:
It's also used for v3 to decide sitePath and won't get changed after document is created. I think let document carry it is better.

---
In reply to: [296725063](https://github.com/dotnet/docfx/pull/4789#discussion_r296725063) [](ancestors = 296725063)

Reply:
sitePath is calculated during documentation creation, no need document to carry it?